homeassistant/components/wirelesstag/sensor.py

```diff
@@ -14,16 +26,45 @@
 _LOGGER = logging.getLogger(__name__)
 
 SENSOR_TEMPERATURE = "temperature"
+SENSOR_AMBIENT_TEMPERATURE = "ambient_temperature"
 SENSOR_HUMIDITY = "humidity"
 SENSOR_MOISTURE = "moisture"
 SENSOR_LIGHT = "light"
 
-SENSOR_TYPES = [SENSOR_TEMPERATURE, SENSOR_HUMIDITY, SENSOR_MOISTURE, SENSOR_LIGHT]
+SENSOR_TYPES: dict[str, SensorEntityDescription] = {
+    SENSOR_TEMPERATURE: SensorEntityDescription(
+        key=SENSOR_TEMPERATURE,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SENSOR_AMBIENT_TEMPERATURE: SensorEntityDescription(
+        key=SENSOR_AMBIENT_TEMPERATURE,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SENSOR_HUMIDITY: SensorEntityDescription(
+        key=SENSOR_HUMIDITY,
+        device_class=DEVICE_CLASS_HUMIDITY,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SENSOR_MOISTURE: SensorEntityDescription(
+        key=SENSOR_MOISTURE,
+        device_class=SENSOR_MOISTURE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SENSOR_LIGHT: SensorEntityDescription(
+        key=SENSOR_LIGHT,
+        device_class=DEVICE_CLASS_ILLUMINANCE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+}
+
+SENSOR_KEYS: list[str] = list(SENSOR_TYPES.keys())
```

Comment:
Copying a dict to a list copies the keys by default.
```suggestion
SENSOR_KEYS: list[str] = list(SENSOR_TYPES)
```